app/spec/lib/bk/compat/snapshots/examples/harness/ci-stage-full.yaml.snap

```diff
@@ -4,10 +4,24 @@ steps:
   key: stage1
   steps:
   - commands:
-    - ' # step [] {:type=>"Background", :name=>"postgres-dependency", :identifier=>"postgresdependency",
-      :spec=>{"connectorRef"=>"myDockerHubConnector", "image"=>"postgres:10.8", "shell"=>"Sh",
-      "envVariables"=>{"POSTGRES_USER"=>"postgres", "POSTGRES_PASSWORD"=>"<+secrets.getValue(\"DbPasswordSecret\")>",
-      "POSTGRES_DB"=>"postgres"}}} not implemented yet :('
+    - "# to make background services work, add the following composefile to your repository"
+    - "# configure their command to force a particular shell in services"
+    - |-
+      cat > compose.yaml << EOF
+      ---
+      services:
+        postgresdependency:
+          image: postgres:10.8
+          environment:
+            POSTGRES_USER: postgres
+            POSTGRES_PASSWORD: <+secrets.getValue("DbPasswordSecret")>
+            POSTGRES_DB: postgres
+
+      EOF
+    plugins:
+    - docker-compose#v5.0.0:
```

Comment:
We should probably bump this given we're on `5.4.0` at the moment through `DEFAULT_VERSIONS` in `models/plugin.rb`. Will make many other specs diff though I imagine!

```suggestion
    - docker-compose#v5.4.0:
```